### PR TITLE
[MAT-6708] Service Scaffolding and Docker stuff

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
## MADiE PR

Jira Ticket: [MAT-6708](https://jira.cms.gov/browse/MAT-6708)
(Optional) Related Tickets:

### Summary

Barebones Sinatra service with two endpoints:

1. PUT Generate QRDA Export XML from MADiE Measure, happy path only.
2. GET Health check

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  qrda-export-mongo:
+    image: mongo:6.0
+    restart: always
+    container_name: 'qrda-export-mongo'
+    ports:
+      - "27017:27017"
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=root
+      - MONGO_INITDB_ROOT_PASSWORD=E5press0
+      - MONGO_INITDB_DATABASE=admin
```
Comment (adongare) [inline docker-compose.yml:21]: should these be env variables? e.g. MONGO_INITDB_ROOT_USERNAME=${ROOT_USER_NAME}

**Review by adongare — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (jkotanchik-SB) [reply docker-compose.yml:21]: Yea, that's just there to get it working quickly on local. Need to work with DevOps to figure out specifics.
